doc/programmers/maintenance.rst

```diff
@@ -2,16 +2,23 @@ Maintenance
 ===========
 
 Description and how-to for maintenance operations.
-Some of the maintenance scripts have been moved to the `pcmsolvermeta
-repository <https://gitlab.com/PCMSolver/pcmsolvermeta>`_
 
-Branching Model and Release Process
------------------------------------
+Branching Model
+---------------
 
-.. warning::
-   **Incomplete or outdated information!**
+.. note::
+
+  All PRs goes to the master branch
+
+Contributions to the code can be divided into three categories: bug fixes, new features, and API breaking new features. All these types of contributions should in principle go into the master branch, through a pull request on `github <https://github.com/PCMSolver/pcmsolver>`_. Only in special cases, typically bug fixes that are not relevant for the master branch, PRs can be directed towards other (release) branches.
+
+A branch is created for every minor release, based on the master branch: ``release/v1.0``, ``release/v1.1``, ``release/v1.2`` etc.
```

Comment:
Yep, it's only a repo. There are some pretty large files there though. I suggest you use [Git LFS](https://git-lfs.github.com/) so that GitHub doesn't complain or block the push.

Reply:
I would remove the "Only in special cases ..." paragraph. A bug fix PR can be opened towards any target branch, since the target branch can be changed later. Also, if it goes to `master` and it is relevant for a release branch, it can be cherry-picked. Indeed that seems to be the way it should be done.

For the creation of release branches, I agree. Rephrase such that it is clear that we branch off when we're nearing a stable release and not whenever.